tests/end2end/Topics-test.js

```diff
@@ -469,4 +469,17 @@ describe('Topics', function () {
         expect(browser.getText(".label")).to.equal(labelName);
     });
 
+    it('add label to topic and check if topic is displayed in topic tab of meeting series', function() {
+        let labelName = 'testLabel';
```

Comment:
Unchanged variables should be marked as constants. So here it would be better to write `const labelName = 'testLabel'`. Since this is an indication for future maintainers and improves the code quality.

Obviously, this is not a merge-blocker. Instead just an advice for the future ;-)